certbot-apache/certbot_apache/tests/configurator_test.py

```diff
@@ -1,4 +1,4 @@
-# pylint: disable=too-many-public-methods
+# pylint: disable=too-many-public-methods, protected-access
```

Comment:
nit: We shouldn't do this at the top level of the file. I think we should either:

1. Make a `_call_create_vhost` method of `AugeasVhostTest` that just forwards all args to `_create_vhost`. In the body of this method, you can have the disabling comment.
2. Place the comment in the body of the `AugeasVhostTest` class like so:

```
class blah(object):
   """docstring"""
    # pylint: disable=protected-access
```